### PR TITLE
Fix bug reading Finnish BERT model and miscellaneous cleanup/bug-fixes.

### PR DESCRIPTION
**Pull request by eusdougc**
This pull request includes a fix for issue #367 and an update to the unit tests to make them run more reliably.  I also a change that should ease compatibility with Keras for versions 2.3.0 and later.

### CODE REVIEW

File: dlpy/model_conversion/write_keras_model_parm.py [hunk 1]
```diff
@@ -35,7 +35,10 @@
 try:
     from keras.engine.topology import preprocess_weights_for_loading
 except ImportError:
-    from keras.engine.saving import preprocess_weights_for_loading
+    try:
+        from keras.engine.saving import preprocess_weights_for_loading
+    except:
```
Comment (kesmit13) [inline dlpy/model_conversion/write_keras_model_parm.py:40]: You should make this only catch ImportError

File: dlpy/tests/test_model.py [hunk 2]
```diff
@@ -90,7 +90,7 @@ def test_model1(self):
                                casout={'name': 'eee', 'replace': True},
                                path=path)
 
-        r = model1.fit(data='eee', inputs='_image_', target='_label_', lr=0.001)
+        r = model1.fit(data='eee', inputs='_image_', target='_label_', lr=1e-5)
```
Comment (kesmit13) [inline dlpy/tests/test_model.py:93]: This value is used in lots of places. Should it be defined as a module or package variable?

File: dlpy/model_conversion/write_keras_model_parm.py (hunk 1 above)
Comment (eusdougc) [reply dlpy/model_conversion/write_keras_model_parm.py]: Good catch - I meant to condition that on ImportError but overlooked it.  My latest update corrects this oversight.

File: dlpy/tests/test_model.py (hunk 2 above)
Comment (eusdougc) [reply dlpy/tests/test_model.py]: I defined safe_learning_rate as a class variable for this test in my latest update.  I reran this unit test and everything still seems stable.

**Pull request merged**